scripts/debug_run.py

```diff
@@ -79,9 +100,22 @@ def grab_graphs(args):
       fd.write(report)
 
 
+def create_temp_folder():
+  seq = -1
+  while True:
+    dir_name = 'debug_run-{}-{}'.format(socket.gethostname(), getpass.getuser())
```

Comment:
this is constant, better to move outside the loop?

Reply:
Makes sense!

Reply:
Actually no :smile: ... it is not constant. We might append.
We could keep a copy outside of the loop, but given this is not critical, IMHO the code reads better this way.